src/Mvc/Mvc.Core/test/Formatters/JsonInputFormatterTestBase.cs

```diff
@@ -194,8 +195,8 @@ protected async Task ReadAsync_ReadsValidArray_AsList(Type requestedType)
 
             // Assert
             Assert.False(result.HasError);
-            var integers = Assert.IsType<List<int>>(result.Model);
-            Assert.Equal(new int[] { 0, 23, 300 }, integers);
```

Comment:
What motivated this change?

Reply:
Good one. I thought I'd bring it up - @layomia was there a reason System.Text.Json returns a type that [wraps `List<T>` ](https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultEnumerableConverter.cs#L7) rather than just returning a `List<T>`? Not that it ultimately matters, but if I remember correctly we agreed on just doing a `List<T>` like Json.NET does.

Reply:
Also cc @steveharter